src/AppBundle/Tests/Controller/AdmissionControllerTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace AppBundle\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class AdmissionControllerTest extends WebTestCase
+{
+    public function testShow()
+    {
+    }
+
```

Comment:
Ska odd lag en issue på test for visning av opptakssida? Veldig fin oppgave som innledning t testing :D